### PR TITLE
Feat/feedback test --run-build

### PR DESCRIPTION
**Pull request by jokomyn**
changelog:
- behaviour save and update member
- seleceted member on order page
- slicing sales recapitulation

### CODE REVIEW

**Comment by akarintitech** [comment]
# SonarQube Code Analytics 
## Quality Gate Status: OK

passed

[Sonar report](https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=17) (https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=17)

## Additional information
*The following metrics might not affect the Quality Gate status but improving them will improve your project code quality.*

## Issues
- Bugs: security_a [0 Bugs](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=BUG&pullRequest=17)

- Vulnerabilities: security_a [0 Vulnerabilities](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=VULNERABILITY&pullRequest=17)

- Code Smells: security_a [0 Code Smells](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=CODE_SMELL&pullRequest=17)

- Security Hotspots: security_a [0 Security Hotspots](https://sonar.akarinti.tech/security_hotspots?id=dayacore_pos_web&pullRequest=17&inNewCodePeriod=true)



## Coverage and Duplications
- Coverage:  [0.00% Coverage](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&metric=new_coverage&view=list&pullRequest=17)

- Duplications:  [0.00% Duplication](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&metric=new_duplicated_lines_density&view=list&pullRequest=17)

File: src/app/modules/member/page/MemberForm.tsx [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+
+                    {/* <GButton
+                    type='button'
+                    variant='OUTLINED'
+                    className='min-w-[138px] text-sm font-medium border border-[#D0D5DD] text-[#344054]'
+                    
+                    data-testid='close-button'
+                    >
+                        Cancel
+                    </GButton> */}
```
Comment (aldiz1711) [inline src/app/modules/member/page/MemberForm.tsx:292]: bisa di remove ya kalau udah ga dipakai

File: src/app/modules/order/Components/Summary.component.tsx [hunk 2]
```diff
@@ -10,7 +10,7 @@ import Keyboard from "react-simple-keyboard";
 import "react-simple-keyboard/build/css/index.css";
 import { FormText, GButton } from 'src/app/components/Libs';
 import { NumericFormat } from 'react-number-format';
-import { buttonTheme, layoutConfig } from 'src/app/components/Features/MenuList/MenuList.component';
+import { buttonTheme, layoutConfig } from 'src/app/App';
 import Carousel from 'src/app/components/Widgets/Carousel.component';
 
 // import "./index.css";
```
Comment (aldiz1711) [inline src/app/modules/order/Components/Summary.component.tsx:16]: bisa di remove kalau udah ga dipakai

File: src/app/modules/member/components/MemberTable.component.tsx [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+                                value={getInputValue('searchLoyaltyMember')}
+                                onFocus={()=>onFocusInput('searchLoyaltyMember')}
+                                onChange={onChangeInput}
+                            ></FormText>
+                            <GButton
+                            className='flex flex-row justify-center items-center border border-[#CF1E26] text-[#CF1E26] rounded-lg h-[44px]'
+                            variant='UNSTYLED'
+                            data-testid='refresh-button'
+                            >
+                                {/* <RotateRight color="#CF1E26" /> */}
```
Comment (aldiz1711) [inline src/app/modules/member/components/MemberTable.component.tsx:204]: bisa di remove jika sudah tidak dipakai

**Comment by akarintitech** [comment]
# SonarQube Code Analytics 
## Quality Gate Status: OK

passed

[Sonar report](https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=17) (https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=17)

## Additional information
*The following metrics might not affect the Quality Gate status but improving them will improve your project code quality.*

## Issues
- Bugs: security_a [0 Bugs](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=BUG&pullRequest=17)

- Vulnerabilities: security_a [0 Vulnerabilities](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=VULNERABILITY&pullRequest=17)

- Code Smells: security_a [0 Code Smells](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=CODE_SMELL&pullRequest=17)

- Security Hotspots: security_a [0 Security Hotspots](https://sonar.akarinti.tech/security_hotspots?id=dayacore_pos_web&pullRequest=17&inNewCodePeriod=true)



## Coverage and Duplications
- Coverage:  [0.00% Coverage](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&metric=new_coverage&view=list&pullRequest=17)

- Duplications:  [0.00% Duplication](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&metric=new_duplicated_lines_density&view=list&pullRequest=17)

**Comment by akarintitech** [comment]
# SonarQube Code Analytics 
## Quality Gate Status: OK

passed

[Sonar report](https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=17) (https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=17)

## Additional information
*The following metrics might not affect the Quality Gate status but improving them will improve your project code quality.*

## Issues
- Bugs: security_a [0 Bugs](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=BUG&pullRequest=17)

- Vulnerabilities: security_a [0 Vulnerabilities](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=VULNERABILITY&pullRequest=17)

- Code Smells: security_a [0 Code Smells](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=CODE_SMELL&pullRequest=17)

- Security Hotspots: security_a [0 Security Hotspots](https://sonar.akarinti.tech/security_hotspots?id=dayacore_pos_web&pullRequest=17&inNewCodePeriod=true)



## Coverage and Duplications
- Coverage:  [0.00% Coverage](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&metric=new_coverage&view=list&pullRequest=17)

- Duplications:  [0.00% Duplication](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&metric=new_duplicated_lines_density&view=list&pullRequest=17)

**Review by aldiz1711 — APPROVED** [review]
OK

**Pull request merged**